svm/src/account_loader.rs

```diff
@@ -545,17 +498,16 @@ mod tests {
         }
     }
 
-    fn load_accounts_with_fee_and_rent(
+    fn load_accounts_with_features_and_rent(
         tx: Transaction,
         ka: &[TransactionAccount],
```

Comment:
(not you, but could clean up while we're here) 

what is `ka` supposed to be short for?

Reply:
I can clean it up in a separate commit. It used to be for "keyed account"

Reply:
PR here: https://github.com/anza-xyz/agave/pull/1691